### PR TITLE
Update galen-pagedump.js

### PR DESCRIPTION
**Pull request by phystem**
Added spec suggestion for relative width and height of elements

### CODE REVIEW

File: src/main/resources/pagedump/galen-pagedump.js [hunk 1]
```diff
@@ -207,6 +207,24 @@ Suggest.prototype.generateSuggestions = function (items) {
     return "<pre>" + html + "</pre>";
 };
 Suggest.prototype._suggestions = {};
+Suggest.prototype._suggestions.width = function(a, b) {
+           return new Spec(a.name, "width: " + getPercentage(a.area[2], b.area[2]) + "% of " + b.name+"/width");
```
Comment (ishubin) [inline src/main/resources/pagedump/galen-pagedump.js]: Seems reasonable

File: src/main/resources/pagedump/galen-pagedump.js [hunk 2]
```diff
@@ -207,6 +207,24 @@ Suggest.prototype.generateSuggestions = function (items) {
...
+   };
+Suggest.prototype._suggestions.height = function(a, b) {
+          return new Spec(a.name, "height: " + getPercentage(a.area[3], b.area[3]) + "% of " + b.name+"/height");
+   };
+Suggest.prototype._suggestions.widthAgainstHeight = function(a, b) {
+    return new Spec(a.name, "width: " + getPercentage(a.area[2], b.area[3]) + "% of " + b.name+"/height");
```
Comment (ishubin) [inline src/main/resources/pagedump/galen-pagedump.js]: Why would you ever need to check the ration of one object width to a height of another object? I have never heard of such situation and personally never had the need to testing this.

File: src/main/resources/pagedump/galen-pagedump.js [hunk 3]
```diff
@@ -207,6 +207,24 @@ Suggest.prototype.generateSuggestions = function (items) {
...
+};
+Suggest.prototype._suggestions.heightAgainstWidth = function(a, b) {
+    return new Spec(a.name, "height: " + getPercentage(a.area[3], b.area[2]) + "% of " + b.name+"/width");
```
Comment (ishubin) [inline src/main/resources/pagedump/galen-pagedump.js]: Same as above. I don't think this is important to have as a suggested spec. If such situation exists where you do have to test the ratio of two different dimension of two different objects, I still think it pretty rare and exceptional

**Comment by ishubin** [comment]
I would like to accept this pull request, but only without ```widthAgainstHeight``` and ```heightAgainstWidth``` suggestions. The reason is they are pretty rare and I would like to keep the suggestion as clean and useful for the user as possible.

File: src/main/resources/pagedump/galen-pagedump.js (hunk 3 above)
Comment (phystem) [inline src/main/resources/pagedump/galen-pagedump.js]: Ya its so rare and exceptional.In our application we need to check whether the element's width is (twice as or x% of) same element's height..So for that purpose only.

Comment (ishubin) [inline src/main/resources/pagedump/galen-pagedump.js]: Wow, that really is weird :) but lets not make it as a general thing for other users. I would rather have your code for width and height as I think those two are more often used checks.

Comment (phystem) [inline src/main/resources/pagedump/galen-pagedump.js]: ok i will remove it then

**Pull request merged**